syntax = "proto3";

message CreateVMSnapshotRequest {

}
